src/GuppyDataExplorer/ExplorerFilterSetsContext.jsx

```diff
@@ -0,0 +1,56 @@
+import { createContext, useContext, useState } from 'react';
+import PropTypes from 'prop-types';
+import { fetchWithCreds } from '../actions';
+import './typedef';
+
+const FILTER_SET_URL = '/amanuensis/filter-set';
```

Comment:
Same question as above for line 6 here.

Reply:
Please see my comment above: https://github.com/chicagopcdc/data-portal/pull/293#discussion_r771635970